lib/authentication_with_facebook/infrastructure/facebook_auth_repository.dart

```diff
@@ -35,7 +35,7 @@ class FacebookAuthenticationRepository implements IFacebookRepositoryFacade {
       if (user == null) {
         return left(const AuthFailure.userNotFound());
       }
-      
+
```

Comment:
Please remove this new line since this file is not a part of your implementation